src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -1204,6 +1203,42 @@ public void testGetPaymentStatusTransition() throws Exception {
         assertThat(returnValue.getToStatus(), is(equalTo(COMPLETED)));
     }
 
+    //
+    // Program Accounts
+    //
+    @Test
+    public void testGetProgramAccount() throws Exception {
+        String functionality = "getProgramAccount";
+        initMockServer(functionality);
+        HyperwalletAccount returnValue;
+        try {
+            returnValue = client.getProgramAccount("prg-83836cdf-2ce2-4696-8bc5-f1b86077238c", "act-da5795ed-d1c9-4231-9e71-611ab8fe9f66");
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+        List<HyperwalletLink> hyperwalletLinks = new ArrayList<>();
+        HyperwalletLink hyperwalletLink = new HyperwalletLink();
+        hyperwalletLink.setHref(
+                "https://api.sandbox.hyperwallet.com/rest/v4/programs/prg-83836cdf-2ce2-4696-8bc5-f1b86077238c/accounts/act-da5795ed-d1c9-4231-9e71"
+                        + "-611ab8fe9f66");
+        Map<String, String> mapParams = new HashMap<>();
+        mapParams.put("rel", "self");
+        hyperwalletLink.setParams(mapParams);
+        hyperwalletLinks.add(hyperwalletLink);
+
+        assertThat(returnValue.getToken(), is(equalTo("act-da5795ed-d1c9-4231-9e71-611ab8fe9f66")));
+        assertThat(returnValue.getType(), is(equalTo(HyperwalletAccount.EType.FUNDING)));
+        assertThat(returnValue.getCreatedOn(), is(equalTo(dateFormat.parse("2017-10-04T21:19:24 UTC"))));
+        assertThat(returnValue.getEmail(), is(equalTo("8715201615-fundingaccount@hyperwallet.com")));
+        if (returnValue.getLinks() != null) {
```

Comment:
please dont add this condition to check null value when you know the expected value. We should not accommodate the missing data in response.

Reply:
Done . thanks @Rrathinasabapath